torchvision/models/video/resnet.py

```diff
@@ -189,16 +211,22 @@ def __init__(self):
 
 class VideoResNet(nn.Module):
 
-    def __init__(self, block, conv_makers, layers,
-                 stem, num_classes=400,
-                 zero_init_residual=False):
+    def __init__(
+        self,
+        block: Type[Union[BasicBlock, Bottleneck]],
+        conv_makers: List[Type[Union[Conv3DSimple, Conv3DNoTemporal, Conv2Plus1D]]],
+        layers: List[int],
+        stem: Callable[..., nn.Module],
+        num_classes: int = 400,
+        zero_init_residual: bool = False,
+    ) -> None:
         """Generic resnet video generator.
 
         Args:
             block (nn.Module): resnet building block
             conv_makers (list(functions)): generator function for each layer
```

Comment:
Not sure if this change is worth it but in [other places](https://github.com/pytorch/vision/blob/183a722169421c83638e68ee2d8fc5bd3415c4b4/torchvision/models/mobilenetv3.py#L119-L120) we define it fully: 
```suggestion
            conv_makers (List[Type[Union[Conv3DSimple, Conv3DNoTemporal, Conv2Plus1D]]]): generator function for each layer
```

I leave it up to you on whether you will change this.